engine/src/test/java/org/hibernate/search/test/dsl/Coffee.java

```diff
@@ -48,6 +49,8 @@ public void setId(String id) {
 	public void setDescription(String description) { this.description = description; }
 	private String description;
 
+	@Field
+	@NumericField
```

Comment:
`@NumericField` is not necessary here, the field should be detected as numeric automatically.

Reply:
Yes, that was my recollection but I wanted to be on the safe side. Removed.